### PR TITLE
ga-release-5.6.0 notes

### PR DESCRIPTION
**Pull request by ScottSuarez**

### CODE REVIEW

**Comment by ScottSuarez** [comment]
cherry picked 85d5caf9613af657b96d3c2b1cde824dce95a098 for `TestAccSqlDatabaseInstance_basicMSSQL`

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,8 @@
 ## 5.6.0 (Unreleased)
-UNKNOWN CHANGELOG TYPE:
-* Add connection resource of Integration Connectors. ([#16468](https://github.com/hashicorp/terraform-provider-google/pull/16468))
-* Update 5.5.0 CHANGELOG.md in main branch ([#16453](https://github.com/hashicorp/terraform-provider-google/pull/16453))
-* Update CHANGELOG.md ([#16405](https://github.com/hashicorp/terraform-provider-google/pull/16405))
-
 FEATURES:
-* **New Data Source:** `google_backup_dr_management_server` (beta) ([#16423](https://github.com/hashicorp/terraform-provider-google/pull/16423))
-* **New Resource:** `google_compute_instance_settings` ([#16472](https://github.com/hashicorp/terraform-provider-google/pull/16472))
+* **New Resource:** ``google_integration_connectors_connection` ([#16468](https://github.com/hashicorp/terraform-provider-google/pull/16468))
```
Comment (c2thorn) [inline CHANGELOG.md:3]:
```suggestion
* **New Resource:** `google_integration_connectors_connection` ([#16468](https://github.com/hashicorp/terraform-provider-google/pull/16468))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,31 @@
...
+* container: marked `binary_authorization.evaluation_mode` field as computed and optional in `google_container_cluster` resource ([#16466](https://github.com/hashicorp/terraform-provider-google/pull/16466))
+* containeraws: added `admin_groups` to `google_container_aws_cluster` ([#16433](https://github.com/hashicorp/terraform-provider-google/pull/16433))
+* containerazure: added `admin_groups` to `google_container_azure_cluster` ([#16433](https://github.com/hashicorp/terraform-provider-google/pull/16433))
+* dataproc: added support for `instance_flexibility_policy` in `google_dataproc_cluster` ([#16417](https://github.com/hashicorp/terraform-provider-google/pull/16417))
+* dialogflowcx: added `is_default_start_flow` field to `google_dialogflow_cx_flow` resource to allow management of default flow resources via Terraform ([#16441](https://github.com/hashicorp/terraform-provider-google/pull/16441))
+* dialogflowcx: added `is_default_welcome_intent` and `is_default_negative_intent` fields to `google_dialogflow_cx_intent` resource to allow management of default intent resources via Terraform ([#16441](https://github.com/hashicorp/terraform-provider-google/pull/16441))
```
Comment (c2thorn) [inline CHANGELOG.md:14]: could be combined?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,31 @@
...
+* gkehub: added `metrics_gcp_service_account_email` to `google_gke_hub_feature_membership` ([#16433](https://github.com/hashicorp/terraform-provider-google/pull/16433))
+* logging: added `index_configs` field to `logging_bucket_config` resource ([#16437](https://github.com/hashicorp/terraform-provider-google/pull/16437))
+* logging: added `index_configs` field to `logging_project_bucket_config` resource ([#16437](https://github.com/hashicorp/terraform-provider-google/pull/16437))
+* monitoring: added `pings_count`, `user_labels`, and `custom_content_type` fields to `google_monitoring_uptime_check_config` resource ([#16420](https://github.com/hashicorp/terraform-provider-google/pull/16420))
+* spanner: added `autoscaling_config` field to  `google_spanner_instance` ([#16473](https://github.com/hashicorp/terraform-provider-google/pull/16473))
+* sql: update `google_sql_database_instance` to reflect that `data_cache_enabled` works for PostgresQL now, in addition to MYSQL ([#16422](https://github.com/hashicorp/terraform-provider-google/pull/16422))
```
Comment (c2thorn) [inline CHANGELOG.md:21]:
```suggestion
* sql: updated `google_sql_database_instance` to reflect that `data_cache_enabled` works for PostgresQL now, in addition to MYSQL ([#16422](https://github.com/hashicorp/terraform-provider-google/pull/16422))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,31 @@
...
+* sql: added `ssl_mode` field to `google_sql_database_instance` resource ([#16394](https://github.com/hashicorp/terraform-provider-google/pull/16394))
+* vertexai: added `private_service_connect_config` to `google_vertex_ai_index_endpoint` ([#16471](https://github.com/hashicorp/terraform-provider-google/pull/16471))
+* workstations: add `domain_config` field to resource `google_workstations_workstation_cluster` (beta) ([#16464](https://github.com/hashicorp/terraform-provider-google/pull/16464))
```
Comment (c2thorn) [inline CHANGELOG.md:24]:
```suggestion
* workstations: added `domain_config` field to resource `google_workstations_workstation_cluster` (beta) ([#16464](https://github.com/hashicorp/terraform-provider-google/pull/16464))
```

**Comment by ScottSuarez** [comment]
Cherry picked revert of 58a2ff7be53e433931df146526a853935664691c - https://github.com/hashicorp/terraform-provider-google/pull/16502
https://github.com/hashicorp/terraform-provider-google/commits/release-5.6.0

**Review by c2thorn — APPROVED** [review]

**Pull request merged**